pkg/metadata/labels.go

```diff
@@ -29,9 +29,8 @@ const (
 const (
 	NodeIDLabel = LabelKroPrefix + "node-id"
 
-	OwnedLabel           = LabelKroPrefix + "owned"
-	KroVersionLabel      = LabelKroPrefix + "kro-version"
-	ControllerPodIDLabel = LabelKroPrefix + "controller-pod-id"
+	OwnedLabel      = LabelKroPrefix + "owned"
+	KroVersionLabel = LabelKroPrefix + "kro-version"
```

Comment:
@matthchr i saw your comment on Go names intialisms i will address it in a seperate PR for all the kro repository. Looks like my s/Symphony/Kro/ wasn't a good idea :)